src/conditions/check_fields.rs

```diff
@@ -51,6 +54,14 @@ impl CheckFieldsPredicate for EqualsPredicate {
         match event {
             Event::Log(l) => l.get(&self.target).map_or(false, |v| match &self.arg {
                 CheckFieldsPredicateArg::String(s) => s.as_bytes() == v.as_bytes(),
+                CheckFieldsPredicateArg::VecString(ss) => {
+                    for s in ss {
+                        if s.as_bytes() == v.as_bytes() {
+                            return true;
+                        }
+                    }
+                    false
```

Comment:
```suggestion
                    ss.iter().any(|s| s.as_bytes() == v.as_bytes())
```

Might want `into_iter` instead of `iter`, not sure there.

Reply:
Going with `iter`